packages/cli/src/commands/index.ts

```diff
@@ -11,6 +11,7 @@ import info from './info/info';
 import config from './config/config';
 import init from './init';
 import doctor from './doctor';
+import profile from '@react-native-community/cli-hermes';
```

Comment:
can we use named export instead of default one? It will be less breaking when we decided to introduce more Hermes-related commands